### DISCUSSION TITLE
Open more info on its own page to allow companion apps to open a specific entity or let a user load a specific entity

### DISCUSSION THREAD

**Discussion by dshokouhi** [home-assistant/frontend#7428] [Other feature requests]
We were having a discussion over in #devs_frontend about the more info pop-up and how to best trigger it:
https://discord.com/channels/330944238910963714/351047592588869643/768574996448149527

The android app has a new beta feature that integrates with Android 11 [Power Menu](https://companion.home-assistant.io/docs/integrations/android-power-menu), this allows us to assign entities to tiles to let the user quickly control a device without opening the app.  A long press on this icon will take the user to their default page but it would be ideal to display the more info dialog to the user.

https://imgur.com/a/pVyyPt3

The second screenshot in the above link shows a black page which is a bug that was just [fixed](https://github.com/home-assistant/android/pull/1090) however we are still just loading the home page and would ideally show just the more info dialog.

Ideally if we can have something like `https://mydomain.com:8123/lovelace?entity_id=sensor.test` would load just the more info dialog for which ever entity gets sent.

This is the starting recommendation:
https://discord.com/channels/330944238910963714/351047592588869643/768591750226116629

This would really help complete the look and allow this page to always show the most up to date content that the frontend team builds.  Also I am sure others may find it useful as well to directly link to this page from say an actionable notification.

**Comment by dshokouhi** [comment]
Just an update on this but in https://github.com/home-assistant/android/pull/1446 we are now triggering the pop-up using some JS mentioned in the links above

**Reply by dshokouhi** [reply]
Looks like after 2021.6 the code used above is no longer working in the android app

**Reply by BoneheadFraggle** [reply]
Just a heads up, in Android 12, Google changes the way Device Controls works as seen here <https://www.androidpolice.com/2021/06/09/android-12-beta-2-gets-new-shortcuts-for-wallet-and-device-controls/>
It's completely moved away from the power menu and is not as easy to use as in Andriod 11.

**Reply by dshokouhi** [reply]
Although they may have changed how you access it the feature request is independent of the power menu. Users can trigger the more info pop-up in notifications and even Android shortcuts. Just about anywhere our webview implementation can be accessed.
